src/js/components/AppBreadcrumbsComponent.jsx

```diff
@@ -1,71 +0,0 @@
-var React = require("react/addons");
```

Comment:
Huh, weird - this showed up as a rename in the IntelliJ source control pane. Guess it wasn't committed as such, thanks for catching that!